modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of the Jaeger Operator.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+  namespace: default
+EOF
+----
+
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
++
+.Tracing YAML example
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    tracing:
+      sample-rate: "0.1"
```

Comment:
Might as well document the `sample-rate` ,  something like:

Defines sampling probability. With `sample-rate: "0.1"`, approximately 1 in 10 traces will be sampled.